zulipterminal/api_types.py

```diff
@@ -107,11 +107,17 @@ class UpdateMessageFlagsEvent(TypedDict):
     all: bool
 
 
+class UpdateRealmEmojiEvent(TypedDict):
+    type: Literal['realm_emoji']
+    realm_emoji: Dict[str, Any]
```

Comment:
This may expand the scope of this PR too far, but if you wished to do a refactor and define and apply a `TypedDict` for the `Any` part of this in the API, that would be useful. I hope it's the same format for the initial_data and events :)

I'm happy to look into this myself if you'd prefer, just let me know.

Reply:
I have defined `RealmEmojiData` to specify value types of the `Any` part.